dandi/tests/fixtures.py

```diff
@@ -390,6 +395,87 @@ def zarr_dandiset(new_dandiset: SampleDandiset) -> SampleDandiset:
     return new_dandiset
 
 
+@pytest.fixture()
+def create_video_files(tmp_path):
+    video_paths = []
+    import cv2
+
+    video_path = tmp_path / "video_files"
+    video_path.mkdir()
+    for no, ext in enumerate([".avi", ".avi"]):
+        movie_file1 = video_path / f"test1_{no}{ext}"
+        movie_file2 = video_path / f"test2_{no}{ext}"
+        (nf, nx, ny) = (5, 10, 20)
+        writer1 = cv2.VideoWriter(
+            filename=str(movie_file1),
+            apiPreference=None,
+            fourcc=cv2.VideoWriter_fourcc(*"DIVX"),
+            fps=25,
+            frameSize=(ny, nx),
+            params=None,
+        )
+        writer2 = cv2.VideoWriter(
+            filename=str(movie_file2),
+            apiPreference=None,
+            fourcc=cv2.VideoWriter_fourcc(*"DIVX"),
+            fps=25,
+            frameSize=(ny, nx),
+            params=None,
+        )
+        for k in range(nf):
+            writer1.write(np.random.randint(0, 255, (nx, ny, 3)).astype("uint8"))
+            writer2.write(np.random.randint(0, 255, (nx, ny, 3)).astype("uint8"))
+        writer1.release()
+        writer2.release()
+        video_paths.append((movie_file1, movie_file2))
+    return video_paths
+
+
+@pytest.fixture()
+def create_video_nwbfiles(create_video_files):
```

Comment:
I don't see a reason to have this in a separate fixture from `create_video_files`.  I think the code could be decently simplified if they were combined.